cbor-schema/plt.cddl

```diff
@@ -0,0 +1,129 @@
+    ; An optional memo.
+    ? "memo": memo
+}
+
+; We represent token amounts as a decimal fraction.
+; The range and precision may be constrained by the token itself.
+token-amount = decfrac
+
+; A memo can be a byte string of up to 256 bytes.
+; When the memo is CBOR encoded, cbor-memo can be used to indicate that.
+memo = raw-memo / cbor-memo
+
+; A raw memo is a byte string of up to 256 bytes.
+raw-memo = bytes .size (0..256)
+
+; Tag 24 indicates that the contained byte string is CBOR encoded.
+; https://www.rfc-editor.org/rfc/rfc8949.html#name-encoded-cbor-data-item
+cbor-memo = #6.24(raw-memo)
+
+; A destination that can receive and hold tokens.
+; Currently, this can only by a Concordium account address.
+token-receiver = tagged-receiver-account
+
+; We use a subtype of the tagged-address type from
+; https://github.com/BlockchainCommons/Research/blob/master/papers/bcr-2020-009-address.md
+tagged-receiver-account = #6.40307(receiver-account)
+
+receiver-account = { 
+    ; If the info (1) field is present, it must indicate CCD.
+    ? 1: tagged-ccd-coininfo,
+    ; We do not support the type (2) field.
+    ; The data (3) field must be the 32-byte representation of a Concordium address
+    3: bytes .size 32
+}
+
+; We use a subtype of the tagged-coininfo type from
+; https://github.com/BlockchainCommons/Research/blob/master/papers/bcr-2020-007-hdkey.md
+tagged-ccd-coininfo = #6.40305(ccd-coininfo)
+
+ccd-coininfo = { 
+    ; The type (1) field is the [SLIP44](https://github.com/satoshilabs/slips/blob/master/slip-0044.md)
+    ; code for Concordium
+    1: 919 
+    ; The network (2) field is not supported.
+}
+
+; Token governance
+
+; A token governance transaction body. This is a list of token governance operations, executed in
+; order. The transaction is executed in the context of a sender account and a designated token.
+; The sender account must have the necessary permissions to execute the operations.
+token-governance-transaction = [ * token-governance-operation ]
+
+; A token governance operation. This can be a mint, burn, or update-list operation.
+token-governance-operation = token-mint
+    / token-burn
+    / token-update-list
+
+; Mint a specified amount to the sender account.
+token-mint = { 
+    ; The operation type is "mint".
+    "type": "mint",
+    ; The amount of tokens to mint.
+    "amount": token-amount
+}
+
+; Burn a specified amount from the sender account.
+token-burn = {
+    ; The operation type is "burn".
+    "type": "burn",
+    ; The amount of tokens to burn.
+    "amount": token-amount
+}
+
+; Update an allow or a deny list by adding or removing an account from it.
+token-update-list = {
+    ; The operation type is "update-list".
+    "type": "update-list",
+    ; The action to take: "add" or "remove".
+    "action": "add" / "remove",
+    ; The list to update: "allow" or "deny".
+    "list": "allow" / "deny",
+    ; The account to add or remove.
+    "target": token-receiver
+}
```

Comment:
I went for this representation, but arguably a more concise format would be appropriate.